src/app.js

```diff
@@ -72,6 +70,39 @@ module.exports = class App {
         return this;
     }
 
+    _init(module, context) {
+        if (isArray(module)) {
+            for (let i = 0; i < module.length; i++) {
+                try {
+                    // eslint-disable-next-line no-param-reassign
+                    context = this._init(module[i], context);
+                } catch (error) {
+                    console.error('error', error);
+
+                    break;
```

Comment:
`break` case looks broken. If we'll meet an error in the middle of array, we'll return the context, inited by previous module. Is it an expected behaviour?